tool/tsh/mfa.go

```diff
@@ -211,7 +215,15 @@ func (c *mfaAddCommand) run(cf *CLIConf) error {
 		return trace.BadParameter("device name can not be empty")
 	}
 
-	dev, err := c.addDeviceRPC(ctx, tc, c.devName, devType, stdin)
+	if wancli.IsFIDO2Available() && !c.pwdless {
+		answer, err := prompt.PickOne(ctx, os.Stdout, stdin, "Allow passwordless logins", []string{"YES", "NO"})
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		c.pwdless = answer == "yes"
+	}
```

Comment:
IIUC I would be prompted by this any time `tsh` is built with FIDO2 support even if I explicitly provided  `--allow-passwordless=false`. Isn't that kind of an odd/annoying user experience?